substanced/folder/generate/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
I've tried both ways, and in my experience it's better to put them into the package. There would be little or no gain by putting them to sdidev. However there would be drawbacks.
- code management: code that really belongs here, is managed in two repos
- reuse: problem if substanced is used in a different, custom buildout.